test/api/context.py

```diff
@@ -0,0 +1,18 @@
+"""
+Use this to import kaos in all the unit tests in this directory:
+
+```
+from .context import kaos
```

Comment:
Can you add a TODO to change this comment.

(It needs to be changed on all the contexts. I'll do it tomorrow.)